kymatio/scattering3d/backend/backend_skcuda.py

```diff
@@ -46,4 +56,29 @@ def cdgmm3d(A, B):
     output : tensor of the same size as A containing the result of the
              elementwise complex multiplication of  A with B
     """
-    raise NotImplementedError(skcuda_notimplementederror)
+    A, B = A.contiguous(), B.contiguous()
```

Comment:
would kind of prefer to have an error raised if the thing is not contiguous to avoid silent creations of copies

Reply:
It's the same in the 2D case : https://github.com/kymatio/kymatio/blob/master/kymatio/scattering2d/backend/backend_skcuda.py#L248

Reply:
yes, unfortunately.

Keep it like this for now for consistency.